code/_experiments.dm

```diff
@@ -6,13 +6,22 @@
 // EXPERIMENT_MY_COOL_FEATURE
 // - Does something really cool, just so neat, absolutely banging, gaming and chill
 
+// EXPERIMENT_WALLENING_SIGNS
+// - Stop-gap for sign/poster directional related stuff like placing them on south related stuff and discarding everything that's not a northern directional on other signs
+
 #if DM_VERSION < 515
 
 	// You can't X-macro custom names :(
 	#ifdef EXPERIMENT_MY_COOL_FEATURE
 		#warn EXPERIMENT_MY_COOL_FEATURE is only available on 515+
 		#undef EXPERIMENT_MY_COOL_FEATURE
 	#endif
+
+	#ifdef EXPERIMENT_WALLENING_SIGNS
+		#warn EXPERIMENT_WALLENING_SIGNS is only available on 515+
+		#undef EXPERIMENT_WALLENING_SIGNS
+	#endif
+
 #elif defined(UNIT_TESTS)
 	#define EXPERIMENT_MY_COOL_FEATURE
```

Comment:
add your define here so its checked by CI

Reply:
The problem with that is that it will then fail CI Compile Maps because the linters won't be passing, and it's presently not feasible to make it so the linters do pass on production since all the maps would look bland for two months in this interstitial period.

I can do the work to make the linters pass and add it to CI on a project branch after this.